netty/src/test/java/io/grpc/netty/NettyAdaptiveCumulatorTest.java

```diff
@@ -84,6 +86,14 @@ void addInput(ByteBufAllocator alloc, CompositeByteBuf composite, ByteBuf in) {
           composite.addFlattenedComponents(true, in);
         }
       };
+
+      // Throws an error on adding incoming buffer.
+      throwingCumulator = new NettyAdaptiveCumulator(0) {
```

Comment:
nit: normally this sort of thing we'd just do inline at the declaration and not in the `@Before`. This is fine though.

Reply:
I'll get it fixed separately to not delay this.